ios/Internxt.xcodeproj/xcshareddata/xcschemes/Internxt.xcscheme

```diff
@@ -41,7 +41,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
is this intentional?

Reply:
those changes are auto generated from `expo prebuild` command, is not intentional but is not something we can prevent